vm/src/stdlib/mod.rs

```diff
@@ -34,6 +34,7 @@ use crate::pyobject::PyObjectRef;
 pub type StdlibInitFunc = Box<dyn Fn(&VirtualMachine) -> PyObjectRef>;
 
 pub fn get_module_inits() -> HashMap<String, StdlibInitFunc> {
+    #[allow(unused_mut)]
```

Comment:
Why was this added? We're returning an immutable HashMap here right?

Reply:
A little bit downwards there are some `#[cfg]`s that conditionally add modules to the map, and if your target matches neither of those `cfg`s, you get an unused mut warning.

Reply:
Ah, I understand now. Thanks!